src/System.Console/src/System/IO/SyncTextReader.cs

```diff
@@ -11,7 +11,6 @@ namespace System.IO
     internal sealed class SyncTextReader : TextReader
     {
         private readonly TextReader _in;
-        private readonly object _methodLock = new object();
```

Comment:
Personally I don't think it's worth an extra field for that.  If you did want something like that, you could do:
```C#
private object MethodLock { get { return this; } } // appropriate comment
```
and then use MethodLock everything instead of _methodLock.  That said, I'm also fine with it as it is.